src/rct1.c

```diff
@@ -189,3 +189,99 @@ const uint8 gRideCategories[] = {
+	{ 0x00000080, { "ZLOG    " }, 0 },		// RCT1_VEHICLE_TYPE_LOG_CARS
+	{ 0x00000080, { "CIRCUS1 " }, 0 },		// RCT1_VEHICLE_TYPE_CIRCUS_TENT
+	{ 0x00000080, { "GTC     " }, 0 },		// RCT1_VEHICLE_TYPE_GHOST_TRAIN_CARS
+	{ 0x00000080, { "BMSD    " }, 0 },		// RCT1_VEHICLE_TYPE_STEEL_TWISTER_ROLLER_COASTER_TRAIN
+	{ 0x00000080, { "MFT     " }, 0 },		// RCT1_VEHICLE_TYPE_WOODEN_TWISTER_ROLLER_COASTER_TRAIN
+	{ 0x00000080, { "SFRIC1  " }, 0 },		// RCT1_VEHICLE_TYPE_WOODEN_SIDE_FRICTION_CARS
+	{ 0x00000080, { "VCR     " }, 0 },		// RCT1_VEHICLE_TYPE_VINTAGE_CARS
+	{ 0x00000080, { "NRL2    " }, 0 },		// RCT1_VEHICLE_TYPE_STEAM_TRAIN_COVERED_CARS
+	{ 0x00000080, { "BMSU    " }, 0 },		// RCT1_VEHICLE_TYPE_STAND_UP_STEEL_TWISTER_ROLLER_COASTER_TRAIN
+	{ 0x00000080, { "BMFL    " }, 0 },		// RCT1_VEHICLE_TYPE_FLOORLESS_STEEL_TWISTER_ROLLER_COASTER_TRAIN
+	{ 0x00000080, { "SMC1    " }, 0 },		// RCT1_VEHICLE_TYPE_STEEL_MOUSE_CARS
+	{ 0x00000080, { "CLIFT2  " }, 0 },		// RCT1_VEHICLE_TYPE_CHAIRLIFT_CARS_ALTERNATIVE
+	{ 0x00000080, { "SMONO   " }, 0 },		// RCT1_VEHICLE_TYPE_SUSPENDED_MONORAIL_TRAIN
+	{ 0x00000080, { "HELICAR " }, 0 },		// RCT1_VEHICLE_TYPE_HELICOPTER_CARS
+	{ 0x00000080, { "VREEL   " }, 0 },		// RCT1_VEHICLE_TYPE_VIRGINIA_REEL_TUBS
+	{ 0x00000080, { "REVCAR  " }, 0 },		// RCT1_VEHICLE_TYPE_REVERSER_CARS
+	{ 0x00000080, { "GOLF1   " }, 0 },		// RCT1_VEHICLE_TYPE_GOLFERS
+	{ 0x00000080, { "SPBOAT  " }, 0 },		// RCT1_VEHICLE_TYPE_RIVER_RIDE_BOATS
+	{ 0x00000080, { "VEKST   " }, 0 },		// RCT1_VEHICLE_TYPE_FLYING_ROLLER_COASTER_TRAIN
+	{ 0x00000080, { "BMRB    " }, 0 },		// RCT1_VEHICLE_TYPE_NON_LOOPING_STEEL_TWISTER_ROLLER_COASTER_TRAIN
+	{ 0x00000080, { "UTCAR   " }, 0 },		// RCT1_VEHICLE_TYPE_HEARTLINE_TWISTER_CARS
+	{ 0x00000080, { "UTCARR  " }, 0 },		// RCT1_VEHICLE_TYPE_HEARTLINE_TWISTER_CARS_REVERSED
+	{ 0x00000080, { "ARRT1   " }, 0 },		// RCT1_VEHICLE_TYPE_RESERVED
+	{ 0x00000080, { "GDROP1  " }, 0 },		// RCT1_VEHICLE_TYPE_ROTODROP_CAR
+	{ 0x00000080, { "FSAUC   " }, 0 },		// RCT1_VEHICLE_TYPE_FLYING_SAUCERS
+	{ 0x00000080, { "CHBUILD " }, 0 },		// RCT1_VEHICLE_TYPE_CROOKED_HOUSE_BUILDING
+	{ 0x00000080, { "MONBK   " }, 0 },		// RCT1_VEHICLE_TYPE_BICYCLES
+	{ 0x00000080, { "ARRT2   " }, 0 },		// RCT1_VEHICLE_TYPE_HYPERCOASTER_TRAIN
+	{ 0x00000080, { "NEMT    " }, 0 },		// RCT1_VEHICLE_TYPE_4_ACROSS_INVERTED_COASTER_TRAIN
+	{ 0x00000080, { "CSTBOAT " }, 0 },		// RCT1_VEHICLE_TYPE_WATER_COASTER_BOATS
+	{ 0x00000080, { "SLCFO   " }, 0 },		// RCT1_VEHICLE_TYPE_FACEOFF_CARS
+	{ 0x00000080, { "JSKI    " }, 0 },		// RCT1_VEHICLE_TYPE_JET_SKIS
+	{ 0x00000080, { "RFTBOAT " }, 0 },		// RCT1_VEHICLE_TYPE_RAFT_BOATS
+	{ 0x00000080, { "AML1    " }, 0 },		// RCT1_VEHICLE_TYPE_AMERICAN_STYLE_STEAM_TRAIN
+	{ 0x00000080, { "THCAR   " }, 0 },		// RCT1_VEHICLE_TYPE_AIR_POWERED_COASTER_TRAIN
+	{ 0x00000080, { "IVMC1   " }, 0 },		// RCT1_VEHICLE_TYPE_SUSPENDED_WILD_MOUSE_CARS
+	{ 0x00000080, { "ENTERP  " }, 0 },		// RCT1_VEHICLE_TYPE_ENTERPRISE_WHEEL
+};
```

Comment:
This data is already in src/rct1/tables.cpp, would be good if you could use that instead.